src/rgw/rgw_admin.cc

```diff
@@ -156,7 +156,8 @@ void usage()
   cout << "  usage trim                 trim usage (by user, date range)\n";
   cout << "  gc list                    dump expired garbage collection objects (specify\n";
   cout << "                             --include-all to list all entries, including unexpired)\n";
-  cout << "  gc process                 manually process garbage\n";
+  cout << "  gc process                 manually process garbage (specify\n";
+  cout << "                             --include-all to process all entries, including unexpired)\n";
```

Comment:
@yaozongyou need also to update the template at `src/test/cli/radosgw-admin/help.t`

Reply:
the help.t has been already updated, see the last file diffs.